### PR TITLE
Feat/academy part 1

### PR DESCRIPTION
**Pull request by MdechampG**
This PR aims to add to the OKP4 Documentation the first part of the OKP4 Academy.
The pages are for the moment hidden.

## Summary by CodeRabbit

- **Documentation**
  - Introduced a comprehensive guide on managing rules within the OKP4 protocol in `docs/academy/rules.md`.

- **New Features**
  - Implemented a reading time estimation feature for MDX documents.

- **Configuration**
  - Updated markdown linter settings to allow emphasis in headings.

- **UI Enhancements**
  - Added new sidebar category "Academy TEST" for better navigation.
  - Introduced a CSS rule to hide elements with the `.hidden` class.

### CODE REVIEW

File: docs/academy/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    <div class="col">
+      <div class="card card-body h-100 d-flex flex-column">
+        <a href="/modules/logic" class="card-title card-link stretched-link">
+          <h2>Logic</h2>
+        </a>
+        <p class="card-text">Discover the heart of Logical interpretation in the protocol.</p>
+      </div>
+    </div>
+  </div>
+</div>
```
[the remark this answers is not in the document]
Comment (ccamel) [reply docs/academy/overview.mdx:108]: For your information, there's no need to check internal links. The CI takes care of this itself and causes the build to fail if there is a broken link.

**Comment by ccamel** [comment]
@MdechampG I took the opportunity to introduce a plugin that calculates reading time based on content (during the build of the doc).

in the markdow, before:

```
*Reading time: 14 min*
```

after:

```
<i>Reading time: {readingTime} min</i>
```

results in:

image

**Review by ccamel — APPROVED** [review]
Thanks so much :+1:

File: docs/academy/rules.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+?- user_role(bob, Role), access_rule(Role, confidential_data, Time).
+```
+
+In this Prolog example:
+We define roles (admin, user, guest) and associate users with roles.
+Access rules are defined based on roles and, in some cases, temporal constraints. For instance, admins have unrestricted access to any dataset, users have access before 18.00, and guests have access to the public_data dataset without temporal constraints.
+We define two datasets (confidential_data and public_data).
+The current time is simulated to illustrate temporal constraints.
+Example queries demonstrate how Prolog can be used to check if a user has access to a specific dataset at the current time based on the defined rules.
+Note: The OKP4 SDK contains templates of governance rules that you can use and adapt to your needs.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/rules.md:243]: Thank you but we don't have the content ready yet. We will add the hyperlink later.

**Review by MathieuConte — APPROVED** [review]

**Review by JulienMassonnat — APPROVED** [review]

**Pull request merged**